### PR TITLE
Don't plot pulse event table by default

### LINKED ISSUE #4630: Don't plot pulse table by default
### What is the expected enhancement?
Currently when plotting a plot schedule with `Schedule.draw` the option `table` is true by default. This should be switched to false.

### PR DESCRIPTION
**Pull request by taalexander**
### Summary
Closes #4630. Disables plotting of the event table by default.

### CODE REVIEW

File: releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+prelude: >
+    Replace this text with content to appear at the top of the section for this
```
Comment (nkanazawa1989) [inline releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml:3]: why those sentences are still here?

Comment (taalexander) [reply releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml]: Fixed in 978a52180

File: releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    Schedule plotting with :py:class:meth:`qiskit.pulse.Schedule.draw` will no
```
Comment (nkanazawa1989) [inline releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml:4]: looks like `:py:class:meth:` causes a warning at docs build. perhaps [`classmethod`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py-classmethod)?

Comment (taalexander) [reply releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml]: Hopefully is fixed now.

**Comment by nkanazawa1989** [comment]
Looks good to me!

**Pull request merged**